silq/pulses/pulse_sequences.py

```diff
@@ -761,7 +1382,10 @@ def __init__(self, pulses=[], **kwargs):
             'NMR_pulses': ['NMR_pulse'],
             'pre_delay': 5e-3,
             'inter_delay': 1e-3,
-            'post_delay': 2e-3}
+            'post_delay': 2e-3,
+            'final_phase': 0,
+            'artificial_frequency': 0
+        }
```

Comment:
what are these for?

Reply:
final pulse phase and artificial frequency to modulate final pulse phase. Ideally these descriptions go into the docstring

Reply:
yeah, agree, docstring would be good